util/key.go

```diff
@@ -35,14 +37,23 @@ func GetCookieKey() (string, error) {
 		var file *os.File
 		var err error
 
-		if file, err = os.OpenFile("config/config-init", os.O_APPEND|os.O_WRONLY, 0644); err != nil {
+		config.CookieKey = encryptcookie.GenerateKey()
+		log.Println("Generated new Cookie Key: ", config.CookieKey)
```

Comment:
there shouldnt be a reason to log the cookie key itself since its not really useful to the user anyways.